versions.yaml

```diff
@@ -56,26 +56,6 @@ versions:
     - tests/functional_tests/running_test.yaml
     - tests/functional_tests/apache_exporter_test.yaml
 
-  # WordPress 4 with PHP 7.3-apache on Debian 9
```

Comment:
WordPress 5 with PHP 7.3-fpm on Debian 9 section can also be deleted with corresponding folder

Reply:
Deleted #WordPress 5 with PHP 7.3-fpm on Debian 9 section in vesrions.yaml and deleted 5/php7/debian9/fpm directory